<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shader Visualizer</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Shaders:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="defaultShaderButton">
        <property name="text">
         <string>Default</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shader1Button">
        <property name="text">
         <string>Metal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="shader2Button">
        <property name="text">
         <string>Glass</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Shapes:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sphereButton">
        <property name="text">
         <string>Sphere</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cubeButton">
        <property name="text">
         <string>Cube</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cylinderTreeButton">
        <property name="text">
         <string>Cylinder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="coneTreeButton">
        <property name="text">
         <string>Cone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="sliderGrid" columnstretch="2,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <property name="horizontalSpacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="recursionsLabel">
          <property name="text">
           <string>Number of recursions:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="angleTextbox"/>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="leafSizeTextbox"/>
        </item>
        <item row="1" column="0">
         <widget class="QSlider" name="recursionsSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSlider" name="angleSlider">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="leafSizeLabel">
          <property name="text">
           <string>Leaf Size:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="angleLabel">
          <property name="text">
           <string>Angle (degrees)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="recursionsTextbox">
          <property name="maxLength">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QSlider" name="leafSizeSlider">
          <property name="maximum">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="seasonLabel">
        <property name="text">
         <string>Season:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="summerRadioButton">
        <property name="text">
         <string>Summer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fallRadioButton">
        <property name="text">
         <string>Fall</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="winterRadioButton">
        <property name="text">
         <string>Winter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="springRadioButton">
        <property name="text">
         <string>Spring</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Animations:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="animNoneButton">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="animMoveButton">
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="animScaleButton">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="animScaleMoveButton">
        <property name="text">
         <string>Move + Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="animRotate1Button">
        <property name="text">
         <string>Rotate1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="animRotate2Button">
        <property name="text">
         <string>Rotate2</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Options:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Wireframe:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="wireframeCheckbox">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>Enabled</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Use Vertex Shader</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="tipsButton">
        <property name="text">
         <string>Tips</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="uniformContainerUI"/>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
